libs/minizip/Makefile

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minizip
+PKG_VERSION:=2.9.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/nmoinvaz/minizip/archive/
+PKG_HASH:=f64b2d228a03673a8ec36a53402e2108437226fd9170d49b7f0c0bca94708b93
+
+PKG_MAINTAINER:=David Woodhouse <dwmw2@infradead.org>
+PKG_LICENSE:=Zlib
+PKG_LICENSE_FILES:=LICENSE
+
+CMAKE_INSTALL:=1
```

Comment:
Erm CMAKE_INSTALL is PKG_INSTALL + InstallDev handling.

You're including a custom InstallDev section. Either that should be removed or CMAKE_INSTALL (PKG_INSTALL is already default).

Reply:
I don't think I need the custom InstallDev section. Will drop it.